### ISSUE TITLE
Problem with "python -m pain001" in Windows

### ISSUE THREAD

**Issue by goral74** [sebastienrousseau/pain001#43]
Hi!
Have some problems

F:\Python\Pythonvirtual\newenv\pain001>python -m pain001
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\Alex\AppData\Local\Programs\Python\Python312\Lib\site-packages\pain001\__main__.py", line 34, in <module>
    from pain001.core.core import process_files
  File "C:\Users\Alex\AppData\Local\Programs\Python\Python312\Lib\site-packages\pain001\core\core.py", line 24, in <module>
    from pain001.csv.load_csv_data import load_csv_data
ModuleNotFoundError: No module named 'pain001.csv'

**Comment by subudas** [comment]
Even i am getting the  same error

  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\subramaniyam.das\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\pain001\__main__.py", line 34, in <module>     
    from pain001.core.core import process_files
  File "C:\Users\subramaniyam.das\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\pain001\core\core.py", line 24, in <module>    
    from pain001.csv.load_csv_data import load_csv_data
ModuleNotFoundError: No module named 'pain001.csv'

**Comment by moizmuhammad** [comment]
I solved this problem by first downloading the source files from Github and comparing it with the folders found on my installation path (../site-packages/pain001/..). I then copied the missing folders to installation path. In my case following 3 folders were missing (xml, csv, db). 

I think the compiled wheel package (version 0.0.24) doesn't includes them.

Hopes this may help in resolving the issue.
